ops/terraform/services/pipeline/modules/bfd_pipeline_slis/lambda_src/cw_metrics.py

```diff
@@ -0,0 +1,196 @@
+import operator
+from dataclasses import dataclass, field
+from datetime import datetime, timedelta
+from functools import reduce
+from itertools import chain, combinations
+from typing import Any, Tuple, TypeVar
+
+T = TypeVar("T")
+
+
+@dataclass
+class MetricData:
```

Comment:
Yeah this is just a refactor of helper functions into their own separate files. Unfortunately there were a lot of refactors here and code re-organization because the Lambda was getting too big for a single source file.